Sources/Starknet/Data/Transaction/Transaction.swift

```diff
@@ -210,7 +363,86 @@ public struct StarknetL1HandlerTransaction: StarknetTransaction {
     }
 }
 
-public struct StarknetDeclareTransactionV0: StarknetTransaction {
+public struct StarknetDeclareTransactionV3: StarknetDeclareTransaction, StarknetTransactionV3 {
```

Comment:
I think this struct is missing `compiled_class_hash`

Reply:
Thanks, good catch. I will add this field and update transaction decoding tests to include v3s.